packages/data-stores/src/wpcom-request-controls/index.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import wpcomProxyRequest from 'wpcom-proxy-request';
+import debugFactory from 'debug';
+
+type WpcomProxyRequestOptions = Parameters< typeof wpcomProxyRequest >[ 0 ];
+
+const debug = debugFactory( 'data-stores:utils:wpcom-wrapper' );
+
+export const wpcomRequest = ( request: WpcomProxyRequestOptions ) =>
+	( { type: 'WPCOM_REQUEST', request } as const );
```

Comment:
This is now a quite generic module that exposes an action creator for triggering controls, and a controls object to add to your store.

Reply:
Nice, it makes good sense to model this after the pattern used in core.